modules/ximgproc/src/lsc.cpp

```diff
@@ -821,7 +821,7 @@ inline void SuperpixelLSCImpl::PostEnforceLabelConnectivity( int threshold )
 
           (*Stmp).Neighbor.insert( (*Stmp).Neighbor.end(), (*S).Neighbor.begin(), (*S).Neighbor.end() );
 
-          sort( (*Stmp).Neighbor.begin(), (*Stmp).Neighbor.end() );
```

Comment:
Stmp->Neighbor.begin(), Stmp->Neighbor.end()

Reply:
I would like to avoid perfoming random refactorings in between.

This is almost automated change.